geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -0,0 +1,242 @@
+      InternalDataSerializer.writeByteArray(member, out);
+      InternalDataSerializer.writeByteArray(score, out);
+    }
+    InternalDataSerializer.writePrimitiveInt(sizeInBytes, out);
+  }
+
+  @Override
+  public void fromData(DataInput in, DeserializationContext context)
+      throws IOException, ClassNotFoundException {
+    super.fromData(in, context);
+    int size = InternalDataSerializer.readPrimitiveInt(in);
+    members = new Object2ObjectOpenCustomHashMap<>(size, ByteArrays.HASH_STRATEGY);
+    for (int i = 0; i < size; i++) {
+      members.put(InternalDataSerializer.readByteArray(in),
+          InternalDataSerializer.readByteArray(in));
+    }
+    sizeInBytes = InternalDataSerializer.readPrimitiveInt(in);
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (!(o instanceof RedisSortedSet)) {
+      return false;
+    }
+    if (!super.equals(o)) {
+      return false;
+    }
+    RedisSortedSet redisSortedSet = (RedisSortedSet) o;
+    if (members.size() != redisSortedSet.members.size()) {
+      return false;
+    }
+
+    for (Map.Entry<byte[], byte[]> entry : members.entrySet()) {
+      if (!Arrays.equals(redisSortedSet.members.get(entry.getKey()), (entry.getValue()))) {
+        return false;
+      }
+    }
+    return true;
+  }
+
+  @Override
+  public int getDSFID() {
+    return REDIS_SORTED_SET_ID;
+  }
+
+  protected synchronized byte[] memberAdd(byte[] memberToAdd, byte[] scoreToAdd) {
+    byte[] oldScore = members.put(memberToAdd, scoreToAdd);
+    if (oldScore == null) {
+      sizeInBytes += calculateSizeOfFieldValuePair(memberToAdd, scoreToAdd);
+    } else {
+      sizeInBytes += scoreToAdd.length - oldScore.length;
+    }
+    return oldScore;
+  }
+
+  private synchronized void membersAddAll(AddsDeltaInfo addsDeltaInfo) {
+    Iterator<byte[]> iterator = addsDeltaInfo.getAdds().iterator();
+    while (iterator.hasNext()) {
+      byte[] member = iterator.next();
+      byte[] score = iterator.next();
+      sizeInBytes += calculateSizeOfFieldValuePair(member, score);
+      members.put(member, score);
+    }
+  }
+
+
+  private synchronized void membersRemoveAll(RemsDeltaInfo remsDeltaInfo) {
+    for (byte[] member : remsDeltaInfo.getRemoves()) {
+      sizeInBytes -= calculateSizeOfFieldValuePair(member, members.get(member));
+      members.remove(member);
+    }
+  }
+
+  /**
+   * @param region the region this instance is stored in
+   * @param key the name of the set to add to
+   * @param membersToAdd members to add to this set; NOTE this list may by modified by this call
+   * @return the number of members actually added
+   */
+  long zadd(Region<RedisKey, RedisData> region, RedisKey key, List<byte[]> membersToAdd,
+      SortedSetOptions options) {
+    int membersAdded = 0;
+    AddsDeltaInfo deltaInfo = null;
+    Iterator<byte[]> iterator = membersToAdd.iterator();
+
+    while (iterator.hasNext()) {
+      boolean delta = true;
```

Comment:
This variable is never used and can be removed.

Reply:
Slaughtered.